recipes/halo/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "halo" %}
+{% set version = "0.0.28" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9ebf98b94a43f3b68e18c6d74dcb1ea58446b8457ce6fb1b2b4cac8d83733f80
+
+build:
+  number: 0
+  script: '{{ PYTHON }} -m pip install . -vv'
+
+requirements:
+  host:
+    - pip
+    - python
+    - setuptools
+  run:
+    - backports.shutil_get_terminal_size >=1.0.0  # [py<33]
```

Comment:
I think this recipe can be no arch and you can remove this dep, because it isn't really relevant anymore

Reply:
Is this an implicit statement that `conda-forge` doesn't support python2.7 any more? I would be more than happy to hear that, and I can make these changes.

Reply:
Yeah, we haven't gone through the effort of dropping builds automatically (we should!) but we don't support it anymore